spring-cloud-stream-binders/spring-cloud-stream-binder-kafka/src/main/java/org/springframework/cloud/stream/binder/kafka/KafkaMessageChannelBinder.java

```diff
@@ -210,6 +210,7 @@ public Object deserialize(byte[] bytes) throws ZkMarshallingError {
 
 	private static final Set<Object> KAFKA_PRODUCER_PROPERTIES = new SetBuilder()
 			.add(BinderPropertyKeys.MIN_PARTITION_COUNT)
+			.add(BinderPropertyKeys.REQUIRED_GROUPS)
```

Comment:
It's a supported property, in the sense that Kafka can satisfy the contract out of the box without additional changes. Being a global property, we must pass it here, or else the Kafka binder will crash. We will have to pass the `requiredGroups` there, to ensure that Rabbit and Redis work as expected - since the ordering of start after deployment cannot be guaranteed.